### PR TITLE
Issue/#58

### LINKED ISSUE #58: Multiple selection checkboxes akin to mcq

### PR DESCRIPTION
**Pull request by hrajotia**
Fixes #58 

- same as adaptlearning/adapt-contrib-mcq/pull/72

### CODE REVIEW

**Comment by oliverfoster** [comment]
Just as a note for accessibility reasons, it's probably wise to just use checkboxes in the template rather than radio buttons. The gmcq/mcq code already makes checkboxes behave as radio buttons based upon the _selectable attribute, so including radio buttons has little merit in javascript. Using checkboxes alone allows a user to interact in a uniform way with the gmcq/mcq component regardless of the number of selectable items, whereas using a mixture of form elements requires futher explaination as to how a user must interact with it as it switches from tab+shifttab+enter to up+down+space in some browsers. It kind of complicates the accessibility stuff unnecessarily.

If you're open I can make amends here and a pull request to the mcq?

File: templates/gmcq.hbs [hunk 1]
```diff
@@ -3,14 +3,14 @@
     <div class="gmcq-widget component-widget clearfix {{#unless _isEnabled}} disabled {{#if _isComplete}} complete submitted show-user-answer {{#if _isCorrect}}correct{{/if}} {{/if}} {{/unless}}">
         {{#each _items}}
         <div class="gmcq-item component-item {{#unless ../_isEnabled}} {{#if _isCorrect}}correct{{else}}incorrect{{/if}} {{/unless}} item-{{@index}} {{odd @index}}">
-            <input type="checkbox" id="{{../_id}}-{{@index}}"/>
+            <input type="{{#if ../_isRadio}}radio{{else}}checkbox{{/if}}" id="{{../_id}}-{{@index}}"/>
```
Comment (oliverfoster) [inline templates/gmcq.hbs]: please change to just "checkbox" for accessibility

**Comment by oliverfoster** [comment]
+1

**Comment by taylortom** [comment]
+1

**Comment by moloko** [comment]
+1

**Pull request merged**